src/Orleans/Providers/ClientProviderRuntime.cs

```diff
@@ -40,7 +40,7 @@ internal class ClientProviderRuntime : IStreamProviderRuntime
         private readonly Dictionary<Type, Tuple<IGrainExtension, IAddressable>> caoTable;
         private readonly AsyncLock lockable;
 
-        private ClientProviderRuntime(IGrainFactory grainFactory) 
+        public ClientProviderRuntime(IGrainFactory grainFactory) 
```

Comment:
can be internal, sure. it was private, since it was Singleton. will change to internal. the whole class is internal, so it does not matter much.